### PR TITLE
Code refactor and Test cases

### PR DESCRIPTION
**Pull request by chilaraiSxt**
This PR contains
1. Test cases for authentication, sql apis and discovery apis
2. Code refactoring - breaking down examples in separate functions
3. Remove old SQL view APIs as they are now directly handled with SQL APIs  
4. Test cases run with GithubWorkflow

### CODE REVIEW

File: .env.sample [hunk 1]
```diff
@@ -3,4 +3,9 @@ BASEURL_DISCOVERY="https://<base_url>/v2"  # Space and Time Discovery API Endpoi
 USERID="" # UserID required for authentication and authorization
 JOINCODE="" # Space and Time Join Code which can be got from the SxT release team
 SCHEME="ed25519"  # The key scheme or algorithm required for key generation
-PREFIX="" # Optional Prefix Parameter for signature generation
\ No newline at end of file
+PREFIX="" # Optional Prefix Parameter for signature generation
+
+# TEST
```
Comment (henrydaly) [inline .env.sample:8]: Do you mean to add these?

Comment (chilaraiSxt) [reply .env.sample:7]: Right. This isn't required.

Comment (henrydaly) [reply .env.sample:7]: No not the prefix - the test vars below this comment. I think you do actually need the PREFIX

Comment (chilaraiSxt) [reply .env.sample:7]: The test credentials are for running unit tests during the CI check with Github workflow and also, if one needs to run the tests locally

Comment (chilaraiSxt) [reply .env.sample:7]: Actually, I have checked for the Prefix and I haven't used the optional param anywhere

**Review by henrydaly — APPROVED** [review]

**Review by Stephen-Hilton — APPROVED** [review]

**Pull request merged**